assets/component-collection-hero.css

```diff
@@ -16,6 +16,7 @@
   .collection-hero.collection-hero--with-image {
     padding: calc(4rem + var(--page-width-margin)) 0
       calc(4rem + var(--page-width-margin));
+    overflow: hidden;
```

Comment:
This I need to make sure it doesn't "spill" on the next elements: [screenshot](https://screenshot.click/22-01-0h55h-vjs8g.png).

Reply:
Another option could be adding the box-shadow-vertical-offset as padding, like we're doing with sliders and you are doing on mobile, but something we can discuss in the future.